router/pkg/config/config.schema.json

```diff
@@ -1716,9 +1716,125 @@
           "default": false,
           "description": "Block non-persisted Operations. If the value is true, the non-persisted operations are blocked."
         },
+        "complexity_calculation_cache": {
+          "type": "object",
+            "description": "The configuration for the complexity calculation cache. The complexity calculation cache is used to cache the complexity calculation for the queries.",
+            "additionalProperties": false,
+            "properties": {
+              "enabled": {
+                "type": "boolean",
+                "default": true,
+                "description": "Enable the complexity calculation cache. If the value is true, the complexity calculation cache is enabled."
+              },
+              "size": {
+                "type": "integer",
+                "default": 1024,
```

Comment:
Can you make this 10*1024=10240?
Also, can you make planner cache, validation cache, and normalization cache the same default amount?
We've seen with some customers that 1024 is too low and 10*1024 doesn't increase memory usage dramatically.